plutus-tx/src/Language/PlutusTx/Lift/Class.hs

```diff
@@ -337,7 +373,7 @@ compileTypeRep dt@TH.DatatypeInfo{TH.datatypeName=tyName, TH.datatypeVars=tvs} =
 
                             defineDatatype tyName (PLC.Def dtvd datatype) deps
                         pure $ mkTyVar () dtvd
-          in flip runReaderT mempty act
+          in runReaderT act mempty
```

Comment:
I somewhat prefer the other one (IMO `runReaderT` itself should take the argument first!)